mongodconn/mailApi.js

```diff
@@ -0,0 +1,23 @@
+const nodeMailer = require('nodemailer');

```

Comment:
why this file is required here?

Reply:
actually I have done this to notify user after inserting his data